crates/nu-command/tests/commands/ucp.rs

```diff
@@ -841,14 +841,13 @@ fn test_cp_arg_no_clobber() {
         let target = dirs.fixtures.join("cp").join(TEST_HOW_ARE_YOU_SOURCE);
         let target_hash = get_file_hash(target.display());
 
-        let actual = nu!(
-        cwd: dirs.root(),
-        "cp {} {} --no-clobber",
-        src.display(),
-        target.display()
+        let _ = nu!(
+            cwd: dirs.root(),
+            "cp {} {} --no-clobber",
+            src.display(),
+            target.display()
         );
         let after_cp_hash = get_file_hash(target.display());
-        assert!(actual.err.contains("not replacing"));
```

Comment:
Just checking that there is still an error here, but perhaps with a different message? If so, I think you can just update the expected string. Same for the other test.

Reply:
Nope, like i said above there is no message. The error is some http error.

Reply:
Oh that's funny, turns out `uutils` was doing a direct `eprintln` if `-n` encountered an existing file. This got removed, but an error is still returned which is much better.